### DISCUSSION TITLE
Help: Play full track instead of just 30s

### DISCUSSION THREAD

**Discussion by tiendungtsptvn** [KRTirtho/spotube#288] [Q&A]
I'm making a simple app to play full track ( get from spotify api ), but with previewUrl (of Track model) i just can play 30s.
How do you play full track, can you show me the simplest way, thanks.

**Comment by KRTirtho** [comment]
You can use the `spotify_sdk` package from https://pub.dev it has native Spotify playback support which by the way requires premium
The `spotify` package can only be used for getting albums/playlists/user-data etc. stuff not the actual audio stream URL
